core/src/main/scala/org/http4s/client/jdkhttpclient/JdkHttpClient.scala

```diff
@@ -1,17 +1,19 @@
 package org.http4s.client.jdkhttpclient
 
-import cats.ApplicativeError
-import cats.effect._
-import cats.implicits._
-import fs2.interop.reactivestreams._
-import fs2.{Chunk, Stream}
 import java.net.URI
 import java.net.http.HttpRequest.BodyPublishers
 import java.net.http.HttpResponse.BodyHandlers
 import java.net.http.{HttpClient, HttpRequest, HttpResponse}
 import java.nio.ByteBuffer
 import java.util
 import java.util.concurrent.Flow
+
+import cats.ApplicativeError
+import cats.effect._
+import cats.implicits._
+import fs2.concurrent.SignallingRef
+import fs2.interop.reactivestreams._
+import fs2.{Chunk, Stream}
```

Comment:
Not a big deal, but I generally try to keep these alphabetized, since all editors can do it easily.